pgx-version-updater/src/main.rs

```diff
@@ -330,11 +339,7 @@ fn is_not_excluded_dir(entry: &DirEntry) -> bool {
 // Check if a specific DirEntry is named "Cargo.toml"
 fn is_cargo_toml_file(entry: &DirEntry) -> bool {
     let metadata = entry.metadata().expect(
-        format!(
-            "Could not get metadata for: {}",
-            entry.file_name().to_str().unwrap()
-        )
-        .as_str(),
+        format!("Could not get metadata for: {}", entry.file_name().to_string_lossy()).as_str(),
```

Comment:
Yeah, it's usually best to use `to_string_lossy` for reporting errors, since it's not ideal to panic when we report errors, but giving a garbled reply is a pretty clear tell as to what must be fixed.

Reply:
Right, an indicator here was the fact that I'd have to `.unwrap` a `.to_str` on file_name.